src/InfluxDB/Point.php

```diff
@@ -81,26 +88,39 @@ public function __toString()
         return $string;
     }
 
+    /**
+     * @param  array $arr
+     * @return string
+     */
     private function arrayToString(array $arr)
     {
         $strParts = array();
 
         foreach ($arr as $key => $value) {
-            $strParts[] = "{$key}={$value}";
+            $strParts[] = sprintf('%s=%s', $key, $value);
```

Comment:
Yes. Dude, really, there's no difference using **sprintf** in there, that's the point. In this case performance isn't a concern. 

So: sprintf is :+1: both in loop and in **throw**. It's a readability concern.